src/preset_cli/api/clients/superset.py

```diff
@@ -539,7 +590,7 @@ def create_dataset(self, **kwargs: Any) -> Any:
         for column in columns:
             column["column_name"] = column["name"]
             column["groupby"] = True
-            if column["is_dttm"]:
+            if column.get("is_dttm") or column.get("is_date"):
```

Comment:
Did some tests using Superset `1.4` and noticed that it returns `is_date` rather than `is_dttm`.

Reply:
```suggestion
            # Superset 1.4 returns ``is_date`` instead of ``is_dttm``
            if column.get("is_dttm") or column.get("is_date"):
```